orangecontrib/text/widgets/tests/test_owpreprocess.py

```diff
@@ -27,6 +27,16 @@ def test_outputs(self):
         self.assertIsNot(pp_data, self.corpus)
         self.assertNotEqual(pp_data, self.corpus)
 
+    def test_previews(self):
+        self.send_signal(self.widget.Inputs.corpus, self.corpus)
+        self.get_output(self.widget.Outputs.corpus)
```

Comment:
This doesn't look too elegant, but it does assure the output. Any nicer ways to do it?